test/Libraries/RevitTestServices/RevitNodeTestBase.cs

```diff
@@ -80,11 +80,13 @@ public virtual void ShutDownTransactionManager()
             }
         }
 
-        private static void SetUpHostUnits()
+#pragma warning disable CS0618
+      private static void SetUpHostUnits()
         {
             BaseUnit.HostApplicationInternalAreaUnit = AreaUnit.SquareFoot;
             BaseUnit.HostApplicationInternalLengthUnit = LengthUnit.DecimalFoot;
             BaseUnit.HostApplicationInternalVolumeUnit = VolumeUnit.CubicFoot;
         }
-    }
+#pragma warning restore CS0618
+   }
```

Comment:
@saintentropy Please chime in here if you can. My understanding is that although these code are for legacy unit nodes, there could still be folks opening up legacy graphs saved in R2021,R20202 etc using legacy unit nodes. In those cases, it would still be good to have them work with the 64 bit ElementId, unless the plan is to remove and auto migrate.